resources/manifest.json

```diff
@@ -3,7 +3,7 @@
 
 	"name": "Buttercup",
 	"description": "Browser extension for Buttercup",
-	"version": "0.14.2",
+	"version": "0.0.0",
```

Comment:
Why are we zero? :D

Reply:
The resources manifest is a template file, which is copied to the dist directory on build. The version is overwritten.